test/src/test/resources/fixtures/extending-types-via-hydration-with-variables-arguments.yml

```diff
@@ -60,8 +60,7 @@ query: |
   }
 variables:
   filter:
-    name:
-      - value
+    name: value
```

Comment:
If you check the definition the value is a `String` and NOT a `List`. The original test had this problem and the test dumper kept it.